component/core/src/main/java/org/exoplatform/social/core/profileproperty/ProfilePropertyServiceImpl.java

```diff
@@ -42,11 +42,14 @@ public class ProfilePropertyServiceImpl implements ProfilePropertyService, Start
 
   private final ProfileSettingStorage                profileSettingStorage;
 
-  private static final String SYNCHRONIZED_DISABLED_PROPERTIES= "synchronizationDisabledProperties";
+  private static final String                        SYNCHRONIZED_DISABLED_PROPERTIES    = "synchronizationDisabledProperties";
 
   protected List<ProfilePropertyDatabaseInitializer> profielPropertyPlugins              = new ArrayList<>();
 
-  private  List<String>                         synchronizedGroupDisabledProperties = new ArrayList<>();
+  private List<String>                               synchronizedGroupDisabledProperties = new ArrayList<>();
+
+  private static final List<String>                  NON_HIDDENABLE_PROPS                =
```

Comment:
We may nee to make those as init params to be able to add others by configuration if needed.

Reply:
OK

Reply:
Done